tests/test_opening_hours.py

```diff
@@ -0,0 +1,38 @@
+from app import app
+from freezegun import freeze_time
+from apistar.test import TestClient
+from idunn.blocks.opening_hour import OpeningHourBlock
+
+def test_opening_hour():
+    """
+    Test that the opening_hours block for a POI located
+    in a different timezone (Moscow) contains the correct 
+    information.
+    """
+    client = TestClient(app)
+    freezer = freeze_time("2018-06-14 9:30:00")
+    freezer.start()
+
+    response = OpeningHourBlock.from_es(
+        {
+            "coord": {
+                "lon": 37.588161523500276,
+                "lat": 55.74831406552745
+            },
+            "properties": {
+                "opening_hours": "Mo-Su 10:00-22:00"
+            }
+        },
+        lang='en'
+    )
+
+    assert response == OpeningHourBlock(
+        status='open',
+        next_transition_time='2018-06-14T22:00:00+00:00',
+        seconds_before_next_transition=45000,
```

Comment:
the test is difficult to read as it is, please add meaningfull comments.

and I don't understand, if it's `9:30` and the shop open as `10:00` how is it open ?

and open question, the `next_transition_time` seems to be formated as UTC (but it's local no ?). should we display it in local, in UTC or in the timezone of the POI ? (I'd say the timezone of the poi)

Reply:
it's supposed to be 9:30 UTC (11:30 in Moscow), so the shop is open.
However the next_transition should be localized in Moscow time (22:00 local time, so the prefix "+00:00" is wrong here). Also the field should be named `next_transition_datetime`.

Another problem that is probably related, the seconds_before_next_transition value is wrong. If it's 11:30 local time, the shop closes at 22:00 (in 10.5 hours = 37800s)

Reply:
done.
I think `next_transition_time` should be displayed in UTC.